integration_test/lit.cfg.py

```diff
@@ -54,7 +54,7 @@
 # directories.
 config.excludes = [
     'Inputs', 'CMakeLists.txt', 'README.txt', 'LICENSE.txt', 'lit.cfg.py',
-    'lit.local.cfg.py'
+    'lit.local.cfg.py', '.circt-test'
```

Comment:
I think this is not necessary, or do we have a plan to consider `.circt-test` as special directory?

Reply:
Without this we get the following "unresolved" message, but I'm open to other ways to avoid it rather than this config change

![image](https://github.com/user-attachments/assets/b5cae170-ff1b-4eb5-9d1b-747a26422324)

Reply:
It looks like llvm-lit is looking into `.*` directories when searching for tests. Ideally we'd be able to generally include `.*` directories in lit, but I'm not sure you can have a regex or glob pattern in `config.excludes`.

Reply:
I think one way is to pass a result directory properly (`-d %T`) to `circt-test` instead of using default `.circt-test`. I think using the default `.circt-test` may cause issues when multiple tests are ran parallely.

Reply:
That's a great idea! `-d %T` sounds like a plan :smile: